include/capstone/ppc.h

```diff
@@ -66,7 +66,7 @@ typedef struct cs_ppc_op {
 	ppc_op_type type;	// operand type
 	union {
 		unsigned int reg;	// register value for REG operand
-		int32_t imm;		// immediate value for IMM operand
+		uint64_t imm;		// immediate value for IMM operand
```

Comment:
`int64_t` sounds better to me, because we still use that for immediates, which can be negative.